java-example.json

```diff
@@ -189,6 +189,44 @@
 					}
 				}
 			}
+		},
+		"Java Swing Example - MultiWindow": {
+			"window": {
+				"id": "MultiWindowJavaSwingExample",
+				"windowType": "native",
+				"path": "$javaExampleJarRoot/MultiWindowJavaSwingExample.jar",
+				"url": "",
+				"arguments": "",
```

Comment:
This example doesn't demonstrate launching different windows under the same process the way the .NET example does. Could we pass an argument that controls whether the JavaFX or the Java Swing example is spawned from the same process?